lib/kasket/visitor.rb

```diff
@@ -135,11 +140,13 @@ def visit_Array(node, *_)
       node.map {|value| visit(value) }
     end
 
-    def visit_Arel_Nodes_Casted(node, *_)
-      case node.val
-      when nil    then nil
-      when String then node.val
-      else quoted(node.val)
+    if ActiveRecord::VERSION::STRING < '5.2'
+        def visit_Arel_Nodes_Casted(node, *_)
+        case node.val
```

Comment:
indentation looks off for this block

Reply:
Weirdly enough, that's what rubocop wants 🤷

Reply:
Looks to be resolved following [the Rubocop updates](https://github.com/zendesk/kasket/compare/0859a42b78132038d0162f4706cf8ca627ab1d4c..f4b80c37fe1d4cb2deb6ed3db4ae7d406032b790#diff-3f02c12d328bdb7a83444581f3dc5742L144) last week (but the comment here isn't showing as 'outdated' after this branch was updated, which seems strange).